src/pages/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { setGlobal } from "reactn";
+import { TutorialApp } from "../components/app";
+
+/*
+  Initialize global state here. If you change the object's structure, such
+  as adding new key value pairs pairs in the object, you will also have to
+  modify the types/shape for the global object in src/global.d.ts to 
+  reflect the global state object you have here.
+*/
+setGlobal({
+  tutorialName: "Hello Data",
+  tutorialStep: 1,
+  selectorDisplay: "block",
```

Comment:
Personally, I think that `selectorDisplay` and `tutorialDisplay` are better being boolean values, and then in the styled components you can state what CSS property they have based on the boolean value. Better to keep CSS properties within css in many cases.